build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id("com.android.application") version "8.6.0-rc01" apply false
```

Comment:
I tried building with an older build of Android Studio, but it complained that 8.6.0-rc01 is too new for it. It seems however that 8.6.0 is already out, and it builds fine with that as well.

Reply:
changed to 8.6